src/app/pages/system/advanced/allowed-addresses/allowed-addresses-form/allowed-addresses-form.component.html

```diff
@@ -11,6 +11,7 @@
         [formArray]="form.get('addresses')"
```

Comment:
Let's also update the label here to make it more clear.

Reply:
@undsoft - what do you mean? 
Which label should I use as an updated value? 😄

Reply:
I mean that it would clearer for the user if instead of `IP Address *` we said `IP Address/Subnet *`